tests/integration/presenting.test.js

```diff
@@ -84,6 +84,8 @@ describe('Verifiable Presentation where both issuer and holder have a Dock DID',
     // Issuer issues credential with id `credId1` to holder with DID `holder1DID`
     cred1 = await issueCredential(issuerKeyDoc, getUnsignedCred(credId1, holder1DID));
 
+    console.log(cred1);
```

Comment:
can we remove the console.logs in this test? if not, can we give some more informative output so that it doesnt appear as garbage when running tests (output wont always be clear when all test are ran due to jest's output system). eg:

```
console.log('VP Test: Issued credential:', cred1);
```

Reply:
*Drive by comment.*
I suggest the first option, remove console logs once the test is working.

Reply:
Done